README.md

```diff
@@ -1,69 +1,45 @@
+MCS Benchmark Data is a set of scripts that can be used to create and execute pipelines that extract, transform and load information about Machine Commonsense benchmarks into the Commonsense Benchmark Ontology.
 
 For more information about the ontology, refer to:
 [Commonsense Benchmark Ontology](https://tetherless-world.github.io/mcs-ontology/docs/index-en.html)
 *Henrique O. Santos, Minor Gordon, Zhicheng Liang, Gretchen Forbush, Deborah L. McGuinness*
+## One-time Setup
 
+### Create the Python virtual environment
 
-## Usage: Adding a Pipeline
+From the current directory:
+    python3 -m venv venv
 
-Adding a new benchmark to the ontology requires adding a pipeline.
-
-### 1. Add the Appropriate Data
 
-Add a folder with the name of the benchmark (snake_case) to the `data` directory. 
-```bash
-cd data
-mkdir <benchmark_name>
-```
-Within that new directory, add the appropriate file/folder structure.
-```bash
-cd <benchmark_name>
-mkdir -p datasets/dev datasets/test datasets/train
-mkdir loaded
-```
+### Activate the virtual environment
 
-Add the benchmark files to the corresponding folder based on their dataset type.
+On Unix:
+    source venv/bin/activate
 
-Copy `metadata.json` from another benchmark's data folder and fill the file with the appropriate information for the new benchmark.
-```bash
-cp ../anli/metadata.json
-```
+On Windows:
 
-Repeat this in the `test_data` directory with smaller subsets of data, if desired. Instead of creating a duplicate metadata file, use symbolic links.
+    venv\Scripts\activate
 
-From root directory, once the test_data directory for this benchmark has been produced:
-```bash
-cd test_data/<benchmark_name>
-ln ../../data/<benchmark_name>/metadata.json
-```
+### Install the dependencies
+    pip install -r requirements.txt
+## Development: Adding a Pipeline
 
-### 2. Create the Benchmark Pipeline Files
+Adding a new benchmark to the ontology requires adding a pipeline.
 
-Make a directory for the new benchmark and copy/create the pipeline files.
 
-```bash
-cd ../mcs_benchmark_data/pipelines
-mkdir <benchmark_name>
-cp ../anli/anli_benchmark_pipeline.py <benchmark_name>_benchmark_pipieline.py
-touch __init__.py <benchmark_name>_transformer.py
-```
-Edit the pipeline files to mimic the respective functionality in the other benchmark pipelines. 
+### 1. Adding a new benchmark
 
-#### Pipeline File
+Activate the virtual environment as above, then run: 
 
-Edit the benchmark pipeline to reflect the new benchmark.
-- Change the class name to `<BenchmarkName>BenchmarkPipeline` in the definition and the call in main
-- Change the class attribute to `ID = "<benchmark_name>"`
-- Change the transformer in the _Pipeline initilaization and the import statement to be called `<BenchmarkName>BenchmarkTransformer`
+    python3.6 -m cli create_benchmark_pipeline --benchmark_name <benchmark_name> [--using-test-data]

```

Comment:
python3 -m `mcs_benchmark_data.cli` ...

Subcommands should be kebab case and not snake case, so create-benchmark-pipeline.